src/components/DataTable/_data-table.scss

```diff
@@ -1 +1,62 @@
 @import '~carbon-components/scss/components/data-table/data-table';
+
+.bx--table-header-label {
+  &-start {
+    span.bx--table-header-label {
+      width: 100%;
+      text-align: start;
+    }
+  }
+  &-center {
+    span.bx--table-header-label {
+      width: 100%;
+      text-align: center;
+    }
+  }
+  &-end {
+    span.bx--table-header-label {
+      width: 100%;
+      text-align: end;
+    }
+  }
+}
+html[dir='rtl'] {
+  td.data-table {
+    &-start {
+      padding-right: 1.25rem;
```

Comment:
Spacing values like padding, margin, etc. should not be hardcoded, but instead use [sass variables provided by carbon](https://www.carbondesignsystem.com/guidelines/spacing#spacing-scale)

Reply:
Now we are able to managed using saas variable i.e.  padding-right: $spacing-09 + $spacing-02;  for 3.25rem

Reply:
Code is updated based on your review comments, please review.